OPHD/States/MapViewStateTurn.cpp

```diff
@@ -544,7 +544,20 @@ void MapViewState::checkNewlyBuiltStructures()
 
 void MapViewState::updateMaintenance()
 {
+	auto sortLambda = [](const Structure* rhs, const Structure* lhs) -> bool
+	{
+		return rhs->integrity() < lhs->integrity();
+	};
```

Comment:
These variables are backwards. You have `rhs` on the left hand side, and `lhs` on the right hand side. That makes reading the function implementation a bit confusing.

I assume the intention is to sort structures such that the lowest integrity is listed first.

Reply:
You are not wrong here. It does in fact sort it correctly but I think that's only because of the parameter order. I'll fix that in a commit here shortly.